examples/citations.links

```diff
@@ -31,11 +31,11 @@ fun header(title) {
 sig suggest : (String, String) ~> ()
 fun suggest(authorPrefix, paperPrefix) client {
   var (authorList, paperList) = completions(authorPrefix, paperPrefix);
-  replaceChildren(
+  domReplaceChildren(
```

Comment:
Eventually we'll want to namespace these properly using some kind of "virtual module", I think. But this namespacing-by-prefixing is fine for now.